ci_support/run_docker_build.sh

```diff
@@ -41,6 +41,9 @@ conda clean --lock
 conda install --yes --quiet conda-forge-build-setup
 source run_conda_forge_build_setup
 
+# Bypass the control-freak run_conda_forge_build_setup and use versions that do work.
+conda install --yes conda-build=1.21.11
```

Comment:
FWIW this is only occurring because the call to `conda inspect linkages` is broken again in later `conda-build` versions. Maybe we could just comment that test until we upgrade to `conda-build` version `2.0.x` where it should be fixed.